test/topology/test_mutation_schema_change.py

```diff
@@ -0,0 +1,150 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Reproducer for a failure during lwt operation due to missing of a column mapping in schema history table.
+"""
+import asyncio
+import logging
+from test.pylib.rest_client import inject_error_one_shot, inject_error
+import pytest
+from cassandra.cluster import Cluster, ConsistencyLevel  # type: ignore # pylint: disable=no-name-in-module
+from cassandra.query import SimpleStatement              # type: ignore # pylint: disable=no-name-in-module
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_mutation_schema_change(manager, random_tables):
+    """
+        Cluster A, B, C
+        create table
+        stop C
+        change schema + do lwt write + change schema
+        stop B
+        start C
+        do lwt write to the same key through C
+    """
+    server_a, server_b, server_c = await manager.running_servers()
+    t = await random_tables.add_table(ncolumns=5, pks=1)
+    manager.driver_close()
+    await inject_error_one_shot(manager.api, server_a.ip_addr,
+                                'raft_server_snapshot_reduce_threshold')
+    await asyncio.sleep(.1)  # wait for io_thread to run
+
+    logger.warning("----- STOPPING C -----")
+    await manager.server_stop_gracefully(server_c.server_id)
+    await manager.driver_connect()
+
+    async with inject_error(manager.api, server_b.ip_addr, 'paxos_error_before_learn'):
+        await t.add_column()
+        ROWS = 1
+        seeds = [t.next_seq() for _ in range(ROWS)]
+        stmt = f"INSERT INTO {t} ({','.join(c.name for c in t.columns)}) " \
+               f"VALUES ({', '.join(['%s'] * len(t.columns))}) "           \
+               f"IF NOT EXISTS"
+        query = SimpleStatement(stmt, consistency_level=ConsistencyLevel.ONE)
+        for seed in seeds:
+            logger.debug("----- FIRST INSERT: %s -----\n", seed)
+            await manager.cql.run_async(query, parameters=[c.val(seed) for c in t.columns])
+        await t.add_column()
+
+    manager.driver_close()
+
+    logger.warning("----- STOPPING B -----")
+    await manager.server_stop_gracefully(server_b.server_id)
+    logger.warning("----- STARTING C -----")
+    await manager.server_start(server_c.server_id)
+
+    await asyncio.sleep(10)
```

Comment:
So did we decide in the end to go with sleeps?

Reply:
please no